cpp/lib/src/util.cc

```diff
@@ -25,15 +25,37 @@
 #include <stdexcept>
 #include <cctype>
 #include <cerrno>
+#include <csignal>
 #include <cstdlib>
 #include <execinfo.h>
 #include <signal.h>
 #include "Compiler.h"
+#include "FileLocker.h"
+#include "FileUtil.h"
 #include "MiscUtil.h"
+#include "SignalUtil.h"
 #include "StringUtil.h"
 #include "TimeUtil.h"
 
 
+static std::string log_path;
+static int log_fd;
```

Comment:
1) What's the difference between log_fd and fd_?
2) It might be clearer to either move these variables to util.h (next to fd_) declaration, or to convert fd_ to a static variable as well

Reply:
Good point.  I eliminated fd_.